yaksh/views.py

```diff
@@ -1071,9 +1072,24 @@ def show_all_questions(request):
             else:
                 context["msg"] = "Please select atleast one question to test"
 
+        if request.POST.get('question_tags'):
+            question_tags = request.POST.getlist("question_tags")
+            search_tags = []
+            for tags in question_tags:
+                search_tags.extend(re.split('[; |, |\*|\n]',tags))
+            search_result = Question.objects.filter(tags__name__in=search_tags,
+                                                    user=user).distinct()
+            context['search_result'] = search_result
+
     questions = Question.objects.filter(user_id=user.id, active=True)
     form = QuestionFilterForm(user=user)
+    user_tags = Question.objects.filter(user=user)\
```

Comment:
I personally feel that the '\' line break should be avoided.

Please store the ```Question.objects.filter(user=user)``` in a variable and then filter further so that you are within the 80 cols